### ISSUE TITLE
Array attribute returned as String 

### ISSUE THREAD

**Issue by chuck-aka-ben** [balderdashy/sails-mysql#141]
My Group Model is returning a string instead of an array

    sails> Group.find().exec(console.log)
    undefined
    sails> null [ { name: 'Super Admins',
      groupType: 'student',
      children: '[1]',
      id: 1,
      createdAt: Tue Aug 19 2014 16:37:07 GMT-0400 (EDT),
      updatedAt: Tue Aug 19 2014 16:37:07 GMT-0400 (EDT) },
    { name: 'Doesn\'t Work',
      groupType: 'student',
      children: '[1]',
      id: 2,
      createdAt: Tue Aug 19 2014 18:28:37 GMT-0400 (EDT),
      updatedAt: Tue Aug 19 2014 18:28:37 GMT-0400 (EDT) } ]

My Group model stores arrays of Student or Laptop IDs like so:

    // Group.js
    ...
    attributes: {
      name: {
        type: 'string',
        required: true
      },

      groupType: {
        type: 'string',
        required: true
      },

      children: {
        type: 'array'
      }
    },
    ...

I was able to get everything working by using a old npm-shrinkwrap.json file I had created that uses sails@0.10.3 and sails-mysql@0.10.5

    // With npm-shrinkwrap.json file

    sails> Group.find().exec(console.log)
    undefined
    sails> null [ { name: 'Super Admins',
      groupType: 'student',
      children: [ 1 ],
      id: 1,
      createdAt: Tue Aug 19 2014 16:37:07 GMT-0400 (EDT),
      updatedAt: Tue Aug 19 2014 16:37:07 GMT-0400 (EDT) },
    { name: 'Doesn\'t Work',
      groupType: 'student',
      children: [ 1 ],
      id: 2,
      createdAt: Tue Aug 19 2014 18:28:37 GMT-0400 (EDT),
      updatedAt: Tue Aug 19 2014 18:28:37 GMT-0400 (EDT) } ]

**Comment by sailsbot** [comment]
Thanks for posting, @chuck-aka-ben.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
